l10n_it_ricevute_bancarie/readme/USAGE.rst

```diff
@@ -14,8 +14,20 @@ I possibili stati della distinta sono: *Bozza*, *Accettata*, *Accreditata*,
 Ad ogni passaggio di stato sarà possibile generare le relative registrazioni
 contabili, le quali verranno riepilogate nella scheda «Contabilità».
 Questa scheda è presente sia sulla distinta che sulle sue righe.
+Queste ultime hanno una vista dedicata per facilitare le
+operazioni sul singolo elemento invece che su tutta la distinta.
 
 La voce di menù 'Presentazione Riba' permette di estrarre le riba fino al
 raggiungimento dell'importo massimo inserito dall'utente. La stessa procedura
 guidata è possibile utilizzarla selezionando i records dalla vista a lista e poi
-cliccare su 'Presentazione Riba' tra le azioni. 
+cliccare su 'Presentazione Riba' tra le azioni.
+
+Nella lista delle fatture è presente una colonna per monitorare l'
+esposizione, cioè l'importo dovuto dal cliente a fronte dell'emissione
+della RiBa non ancora scaduta.
+
+In maniera predefinita la data delle registrazioni dei pagamenti viene
```

Comment:
Qua nella 16.0 avevo messo questa descrizione per indicare tutte le modalità di modifica della data di scadenza:

```
In maniera predefinita la data delle registrazioni dei pagamenti viene
impostata con la data di scadenza della RiBa, ma è possibile modificarla in due momenti:
- durante la creazione del pagamento,
  cliccando su "Segna righe come pagate" o su "Segna coma pagata"
  o usando l'azione "Registrazione Riba a data di scadenza"
  e indicando una data nel campo `Data pagamento`,
- successivamente a pagamento effettivamente avvenuto selezionando la
registrazione dalla vista ed elenco ed eseguendo l'azione "Imposta data
di pagamento RiBa".
```

Reply:
@odooNextev dal blame risulta che quella parte è aggiunta in https://github.com/OCA/l10n-italy/pull/4383 (stiamo backportando anche quella)

Reply:
ok